standard/lexical-structure.md

```diff
@@ -285,18 +291,50 @@ Whitespace
 There are several kinds of ***tokens***: identifiers, keywords, literals, operators, and punctuators. White space and comments are not tokens, though they act as separators for tokens.
 
 ```ANTLR
-token
-    : identifier
-    | keyword
+fragment Token
+    : Simple_Identifier
+    | Any_Keyword
     | Integer_Literal
     | Real_Literal
     | Character_Literal
     | String_Literal
-    | operator_or_punctuator
+    | Any_Operator_Or_Punctuator
+    ;
+
+fragment Any_Keyword
+    : 'abstract'   | 'add'       | 'alias'      | 'as'        | 'ascending'
+    | 'async'      | 'await'     | 'base'       | 'bool'      | 'break'
+    | 'by'         | 'byte'      | 'case'       | 'catch'     | 'char'
+    | 'checked'    | 'class'     | 'const'      | 'continue'  | 'decimal'
+    | DEFAULT      | 'delegate'  | 'descending' | 'do'        | 'double'
+    | 'dynamic'    | 'else'      | 'enum'       | 'equals'    | 'event'
+    | 'explicit'   | 'extern'    | 'false'      | 'finally'   | 'fixed'
+    | 'float'      | 'for'       | 'foreach'    | 'from'      | 'get'
+    | 'global'     | 'goto'      | 'group'      | 'if'        | 'implicit'
+    | 'in'         | 'int'       | 'interface'  | 'internal'  | 'into'
+    | 'is'         | 'join'      | 'let'        | 'lock'      | 'long'
+    | 'nameof'     | 'namespace' | 'on'         | 'orderby'   | 'override'
+    | 'params'     | 'partial'   | 'private'    | 'protected' | 'public'
+    | 'readonly'   | 'ref'       | 'remove'     | 'return'    | 'sbyte'
+    | 'sealed'     | 'select'    | 'set'        | 'short'     | 'sizeof'
+    | 'stackalloc' | 'static'    | 'string'     | 'struct'    | 'switch'
+    | 'this'       | 'throw'     | 'true'       | 'try'       | 'typeof'
+    | 'uint'       | 'ulong'     | 'unchecked'  | 'unsafe'    | 'ushort'
+    | 'using'      | 'value'     | 'var'        | 'virtual'   | 'void'
+    | 'volatile'   | 'where'     | 'while'      | 'yield'
+    ;
+
+fragment Any_Operator_Or_Punctuator
```

Comment:
I seem to remember a reference to Operator_Or_Punctuator in a note, but I can't see a change to that note here.